.github/dependabot.yml

```diff
@@ -26,6 +26,8 @@ updates:
         dependency-type: direct
       - dependency-name: rspec-rails
         dependency-type: direct
+      - dependency-name: mongoid
```

Comment:
Could this be moved to between lines 22 and 23? So far, the list is alphabetical - let's keep it that way!

Reply:
Good point.  I've moved this up the list.